filebeat/tests/system/test_base.py

```diff
@@ -0,0 +1,28 @@
+from filebeat import BaseTest
+import os
+
+"""
+Test for basic object
+"""
+
+
+class Test(BaseTest):
+
+    def test_base(self):
+        """
+        Test if the basic fields exist.
+        """
+        self.render_config_template(
+            path=os.path.abspath(self.working_dir) + "/test.log",
```

Comment:
After dealing with weird path issues lately, I recommend using `os.path.join` over string concat to join paths. Or even just putting the `+ "/test.log"` inside the `abspath()` call would fix the problem because `abspath` normalizes the path that it returns.